models/database.go

```diff
@@ -21,19 +21,16 @@ import (
 	"fmt"
 	"net/url"
 	"strings"
+	"time"
 
-	"github.com/go-sql-driver/mysql"
 	"github.com/lib/pq"
 	"github.com/pkg/errors"
 	"gopkg.in/reform.v1"
 )
 
-// FIXME Re-add created_at/updated_at: https://jira.percona.com/browse/PMM-3350
+var now = time.Now().UTC().Format(time.RFC3339)
```

Comment:
Uh! Let's use a long name there – otherwise, it is _too_ easy to misuse it in `BeforeInsert `, for example.